garage/torch/envs/mujoco_env.py

```diff
@@ -0,0 +1,64 @@
+import os
+from os import path
+
+import mujoco_py
+import numpy as np
+from gym.envs.mujoco import mujoco_env
+
+from garage.torch.core.serializable import Serializable
+
+ENV_ASSET_DIR = os.path.join(os.path.dirname(__file__), 'assets')
+
+
+class MujocoEnv(mujoco_env.MujocoEnv, Serializable):
```

Comment:
Is this necessary? How is this different from the MujocoEnv we have?

Reply:
They both inherit gym.Env and share the most important APIs. But there are also several differences on initialization and behavior control that requires some effort to sync up.
- Different parameters.
Some parameters in base but not in torch, some are vice versa. For the latter ones we need to modify MujocoEnv in base.  e.g torch exposures manual control of `frame_skip`.
- Torch's MujocoEnv inherits Serializable but base MujocoEnv does not. I need some experiments to see if there will be problem. 

Luckily there are now only two mujoco envs supported in torch. I will try working on this issue later.

Reply:
MujocoEnv is removed from torch now.